src/server/mods.cpp

```diff
@@ -98,11 +99,17 @@ void ServerModManager::getModNames(std::vector<std::string> &modlist) const
 
 void ServerModManager::getModsMediaPaths(std::vector<std::string> &paths) const
 {
-	for (const ModSpec &spec : m_sorted_mods) {
-		fs::GetRecursiveDirs(paths, spec.path + DIR_DELIM + "textures");
-		fs::GetRecursiveDirs(paths, spec.path + DIR_DELIM + "sounds");
-		fs::GetRecursiveDirs(paths, spec.path + DIR_DELIM + "media");
-		fs::GetRecursiveDirs(paths, spec.path + DIR_DELIM + "models");
-		fs::GetRecursiveDirs(paths, spec.path + DIR_DELIM + "locale");
-	}
+	std::for_each(m_sorted_mods.rbegin(), m_sorted_mods.rend(),
+			[&paths](const ModSpec &spec) {
+				fs::GetRecursiveDirs(paths,
+						spec.path + DIR_DELIM + "textures");
+				fs::GetRecursiveDirs(
```

Comment:
this is a bit ... ugly

I'd prefer manual iterator usage over this.

Reply:
Done.